src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -119,7 +119,10 @@
   </resheader>
   <data name="ShellBannerNonWindowsPowerShell" xml:space="preserve">
     <value>PowerShell {0}
-Copyright (c) Microsoft Corporation. All rights reserved.</value>
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+To get help type Help or Help followed by a command name, such as: Help Get-Process
```

Comment:
Perhaps instead of giving an example just recommend typing `Help`. the default help topic is about using the help system.

Reply:
I agree that we can simply say:

```
Type `help` to get help.
```